.gitignore

```diff
@@ -4,3 +4,4 @@ build/
 
 /.idea/
 /local.properties
+.DS_Store
```

Comment:
Maybe we can put this in the global ignore file, as it's not a project-specific thing? :grin: